examples/nidcpower/measure-record.py

```diff
@@ -155,6 +157,10 @@ def on_close(event):
 
     print("\nReading values in loop. CTRL+C or Close window to stop.\n")
 
+    # Create a buffer for fetching the values.
+    y_axis = [0]*(record_length*buffer_multiplier)
```

Comment:
Space out the operators in this example: `[0]*(...` --> `[0] * (...`.
Same on lines 188, 190, 194, 203

Reply:
Done. Thanks.